next/components/forms/segments/UserProfile/UserProfileView.tsx

```diff
@@ -82,8 +83,8 @@ const UserProfileView = () => {
           }}
           title={t('profile_detail.modal_title')}
         >
-          <AccountMarkdown content={t('profile_detail.modal_message')} variant="sm" />
-          <p className="mt-6">{t('profile_detail.modal_thanks')}</p>
+          <AccountMarkdown content={t('profile_detail.modal_message')} variant="sm" className={cx("text-center","md:text-left")} />
```

Comment:
why adding cx, here and elsewhere - shouldn't it behave the same without it ?

Reply:
forgot to remove it while i tried some things

Reply:
for better clean code. i always divide classNames to 2 separate params in cx. one are basic classes and second is media queries.